scripts/bootstrap-cluster.sh

```diff
@@ -145,7 +145,7 @@ start_kafka_prometheus_exporter(){
   -v kafka_prometheus_volume:/etc/certs \
   danielqsj/kafka-exporter \
   --kafka.server=kafka:9092 \
-  --web.telemetry-path=/pmetrics \
+  --web.telemetry-path=/metrics \
```

Comment:
Is the metrics endpoint protected with Basic Auth? Otherwise metrics are available to anyone who happens to drive by and query the Container Host IP address space.

Reply:
It is the same as before this PR, metrics were not protected.

Reply:
Allright, maybe we should create an issue to make this visible. Having an open /metrics endpoint is problematic, especially as we went to such great lenghts to add TLS :)

Reply:
Agree :) add TLS just to open up metrics endpoints isn't the nicest thing :P I can't think of a way to secure those endpoints right now though (same is true for the JMX exporter, but on a different port). But let's open a issue indeed and fix it that way